docs/self-managed/components/orchestration-cluster/core-settings/concepts/backups.md

```diff
@@ -0,0 +1,26 @@
+---
+id: backups
+title: "Backups"
+description: "Learn more about backups with the Orchestration Cluster."
+---
+
+When running an orchestration cluster with **Operate** and **Tasklist**, you must configure a snapshot repository in your chosen database:
```

Comment:
```suggestion
When running an orchestration cluster with **Secondary Storage**, you must configure a snapshot repository in your chosen database:
```
Would it make more sense to frame it this way? I don't know to what extent users are familiar with this concept though

Reply:
I'll adjust the wording, and link to the changes with https://github.com/camunda/camunda-docs/pull/6779/files once they are merged.